README.md

```diff
@@ -0,0 +1,123 @@
+# ActiveStorageBase64
+
+Gem used to add support for base64 images for Rails's ActiveStorage.
+
+## Getting Started
+
+In order to get the gem working on your project you just need to add the gem to your project like this:
+```ruby
+gem 'active-storage-base64'
+```
+
+### Prerequisites
+
+The only two prerequisites for using this gem are having Rails version 5.2.0 or higher installed on your project and having ActiveStorage setup properly (for more information on how to do this, check this [Active Storage Overview](https://edgeguides.rubyonrails.org/active_storage_overview.html))
+
+```ruby
+gem 'rails', '5.2.0'
+```
+
+### Installing
+
+In order to use the gem's functionality, you'll need to include the module into your ActiveRecord inheriting class.
+For example:
+```ruby
+class Test < ActiveRecord::Base
+    include ActiveStorageSupport::SupportForBase64
+end
+```
+After you have the module included in your class you'll be able to use the following two helper methods for working with base64 files:
+When you need a single image attached:
+```ruby
+has_one_base64_attached
+```
+and when you need multiple files attached:
+```ruby
+has_many_base64_attached
+```
+These helpers will work just like the `has_one_attached` and `has_many_attached` helper methods from ActiveStorage.
+
+A working example for this, assuming we have a model `Test` with only a picture attached would be:
+```ruby
+class Test < ActiveRecord::Base
```

Comment:
I think this example doesn't add anything having the one below (user with avatar)

Reply:
Okay, I will use only the User example then